dynatrace-oneagent-operator/README.md

```diff
@@ -39,6 +39,27 @@ Generate an API and a PaaS token in your Dynatrace environment.
 
 https://www.dynatrace.com/support/help/reference/dynatrace-concepts/why-do-i-need-an-environment-id/#create-user-generated-access-tokens
 
+#### Apply CustomResourceDefinition
+
+Apply the latest version of the CRD attached to [the latest release](https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest).
+
+##### Kubernetes and OpenShift
+```
+$ kubectl apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/crd.yaml
```

Comment:
For the `dynatrace-oneagent-operator` we only have `kubernetes.yaml` and `openshift.yaml`.

Should we change this here or on the operator repo?

Reply:
We will put the CRD files as release artefacts to the operator releases.
kubernetes.yaml and openshift.yaml are deployment files for the whole operator not just the CRDs.

We can't release it on this repository since we release the dynatrace-operator and the dynatrace-oneagent-operator within the same repository. Linking to `/release/latest` wouldn't work therefore